structuredefinitions/UKCore-Specimen.xml

```diff
@@ -159,6 +169,12 @@
         <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-Practitioner" />
         <targetProfile value="https://fhir.hl7.org.uk/StructureDefinition/UKCore-PractitionerRole" />
       </type>
+      <constraint>
```

Comment:
This possibly needs moving up due to the other changes

Reply:
Ignore that, its ok where it is, but the human value needs updating

Reply:
and the expression?

Reply:
moved expression to extension and rewrote human value